File: answer_1/alstamber_scala/6.scala
object main6{
  def main(args: Array[String]) = {
    val height = readDouble()
    val weight = readDouble()

    val bmi = weight / (height * height)

    println( bmi match {
      case b if b < 18.5 => "yase"
      case b if b < 25.0 => "normal"
      case b if b < 30.0 => "himan1"
      case b if b < 35.0 => "himan2"
      case b if b < 40.0 => "himan3"
      case _ => "himan4"
      }
    )
  }
}

File: answer_1/alstamber_ruby/6.rb
height = gets.to_f
weight = gets.to_f

bmi = weight / (height * height)

if bmi < 18.5
  puts "yase"
elsif bmi < 25.0
  puts "normal"
elsif bmi < 30.0
  puts "himan1"
elsif bmi < 35.0
  puts "himan2"
elsif bmi < 40.0
  puts "himan3"
else
  puts "himan4"
end

